AUTHORS

```diff
@@ -0,0 +1,28 @@
+Contributors include::
```

Comment:
```suggestion
Contributors include:
```

Reply:
Depends. I think it's like this in pytest because they include it in their ReST documentation directly, and that makes it a literal block